CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ full changeset diff at the end of each section.
 
 Current Trunk
 -------------
+- Add a new flag to Directize, `--directize-initial-tables-immutable` which
```

Comment:
When you refer to this argument you don't prefix it with `--pass-arg`.. even though IIUC in practice one always needs that.  How should we document/refer to these flags?

Reply:
Ah, right, this should be prefixed. Fixed in both places in the docs.